drivers/builtin/include/mbedtls/oid.h

```diff
@@ -628,7 +628,7 @@ int mbedtls_oid_get_md_hmac(const mbedtls_asn1_buf *oid, mbedtls_md_type_t *md_h
  */
 int mbedtls_oid_get_md_alg(const mbedtls_asn1_buf *oid, mbedtls_md_type_t *md_alg);
 
-#if !defined(MBEDTLS_X509_REMOVE_INFO)
+#if 1 /* OID_INFO_STRINGS */
```

Comment:
Are we going to keep this `#if 1`? Can't we just remove it?
If so, then there are other occurrences also in `oid.c`.

Reply:
Since oid.c will be reworked at some point, the decision was to replace this macro with 1 instead to preserve information but enable `check_names` to work without reporting errors.

Reply:
We prefer to keep the information in the code of where the "!defined(MBEDTLS_X509_REMOVE_INFO)" were to ease the rework of the oid.c APIs (see https://github.com/Mbed-TLS/mbedtls/issues/9380).

Reply:
We're actually not redesigning the OID API in TF-PSA-Crypto 1.0: it's becoming private. So I've created a separate task for handling the split of the effect of `MBEDTLS_X509_REMOVE_INFO`: https://github.com/Mbed-TLS/mbedtls/issues/10124. Fortunately it seems that none of the OIDs used in crypto need a description any more.